recipes/dftd3-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dftd3-python" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/awvwgk/simple-dftd3/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 79b507a0d892f19222e1403f4c7ebb98ce9eb96b3b8d4821ba5631aa9ac756be
+
+build:
+  number: 0
+  skip: True  # [win or (osx and (python_impl == 'cpython' and py<38))]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson 0.55.*
```

Comment:
What's the reason for this hard meson pin? Maybe add a comment here.

Reply:
0.56.* and 0.57.* were quite broken lately for most of my projects, seems to be fixed with the 0.58.* release now.